packages/rke2-calico/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,12 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,5 +1,8 @@
++annotations:
++  # Operator is installed in tigera-operator ns but calico components are installed in calico-system
```

Comment:
Is this comment accurate? Looks like they're going into rke2-calico ns.

Reply:
Not sure where you see rke2-calico ns. When looking at the pods:
`calico-system     calico-node-f7wtz                                    1/1     Running     0          42m   10.17.3.3       worker2   <none>           <none>`

Reply:
Apologies, I misread the patch!